manifests/database.pp

```diff
@@ -129,20 +132,27 @@
   $database_host_ip                = $zabbix::params::server_database_host_ip,
   $database_charset                = $zabbix::params::server_database_charset,
   $database_collate                = $zabbix::params::server_database_collate,
+  Optional[String[1]] $database_tablespace = $zabbix::params::server_database_tablespace,
```

Comment:
is this change in the PR on purpose or by accident? the tablespace isn't related to the hostinterface/hostinterfaceitem?